apps/hermes/client/js/.eslintrc.js

```diff
@@ -3,4 +3,10 @@ module.exports = {
   parser: "@typescript-eslint/parser",
   plugins: ["@typescript-eslint"],
   extends: ["eslint:recommended", "plugin:@typescript-eslint/recommended"],
+  rules: {
```

Comment:
Please remove this and add eslint-ignore comments to lines where variables are intentionally ignored instead.